README.md

```diff
@@ -249,9 +249,15 @@ First of all, you need a compiler with decent C11 support. The following compile
 - MSVC >= 2015
 - XCode Clang >= 6.0
 
-Pony requires LLVM version 3.9.1.
+When building from sources by default Pony uses LLVM as installed in you system or you may also build LLVM from sources. In either case the supported version of LLVM is 3.9.1 with experimental support for version 6.0.0. Versions other than LLVM 3.9.1 may result in decreased performance or crashes in generated applications.
 
-There is experimental support for building with LLVM 4.0.1, 5.0.1 or 6.0.0, but this may result in decreased performance or crashes in generated applications.
+To build the LLVM sources:
```

Comment:
I would love to see some more guidance on how to compile LLVM and what is necessary to do so in this README.